auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -318,7 +320,6 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
     }
 
     this.isLoading = 1 // truthy
-    this.$route.query.isNumberedCompanyRequest && await this.createNumberedBusiness(this.currentAccountSettings.id)
```

Comment:
This is old code to create a numbered business while loading the business registry page. This is now obsolete -- there is now an explicit button to create a numbered business.